applications/mne_rt_server/mne_rt_server/connectormanager.cpp

```diff
@@ -29,7 +29,7 @@
 * POSSIBILITY OF SUCH DAMAGE.
 *
 *
-* @brief     implementation of the ConnectorManager Class.
+* @brief     Definition of the ConnectorManager Class.
```

Comment:
Must be Declaration

Reply:
actually this is the source file so I think Definition should be fine